workbench-core/swb-common-ui/src/models/User.ts

```diff
@@ -27,12 +27,12 @@ export const researcherUser: UserItem = {
 
 export const adminUser: UserItem = {
   id: 'sample-admin-id',
-  givenName: 'Admin',
+  givenName: 'ITAdmin',
   familyName: 'User',
   email: 'sample.user@example.com',
   avatar: { name: 'user-profile' },
   claims: [],
-  role: 'admin'
+  role: 'itadmin'
```

Comment:
`ITAdmin`?
(that is the spelling used in post-deployment)

Reply:
Based on my (limited) understanding role is lowercase but group is uppercase so I kept letter case to avoid side effects. Can someone shed some light on it?

Reply:
called in the experts...

Reply:
> `role` and `group` are two names for the same thing, a Cognito user group. User management uses the `role` variable while authZ uses `group`
> The value for a user's role should match whatever Cognito group they belong to